lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb

```diff
@@ -14,24 +14,23 @@ class DVCOptions
     attr_accessor :config_cdn_uri
     attr_accessor :events_api_uri
     attr_accessor :bucketing_api_uri
+    attr_accessor :event_request_chunk_size
```

Comment:
it occurs to me that we shouldn't be setting these as accessors; as that way you can override and bypass our clamping after instantiation. Should probably be read only

Reply:
kk will make it read only